feature/bgp/feature.textproto

```diff
@@ -430,6 +430,12 @@ config_path {
 telemetry_path {
   path: "/network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/timers/state/keepalive-interval"
 }
+config_path {
```

Comment:
Can we also update the Config Library code to include this?

Reply:
Done!